tests/acceptance/docker/Makefile

```diff
@@ -122,27 +119,20 @@ test-ocis-feature-s3ng-storage: ## test a ocis feature with s3ng storage, usage:
 	$(MAKE) --no-print-directory testSuite
 
 .PHONY: test-oc10-feature-ocis-storage
-test-oc10-feature-ocis-storage: ## test a oC10 feature with oCIS storage, usage: make ... BEHAT_FEATURE='tests/acceptance/features/apiAuth/cors.feature'
+test-oc10-feature-ocis-storage: ## test a oC10 feature with oCIS storage, usage: make ... BEHAT_FEATURE='tests/acceptance/features/coreApiAuth/webDavAuth.feature'
```

Comment:
Need suggestions on if we want to rename the command `test-oc10-feature-ocis-storage` as it's technically not oc10 feature now but ocis' and what would be the appropriate naming for this? this as similar to what we do on natively installed test `test-acceptance-from-core-api`

Reply:
Maybe we keep using the word "core".
`test-core-feature-ocis-storage`

These are tests that check that the "core" of the API is compatible with (works the same as) ownCloud 10.